zhaquirks/tuya/ts011f_plug.py

```diff
@@ -95,6 +95,73 @@ class Plug(EnchantedDevice):
     }
 
 
+class CircuitBreaker(EnchantedDevice):
+    """Tuya smart circuit breaker with metering (_TZ3000_qeuvnohg) https://www.aliexpress.com/item/1005004747066832.html."""
+
+    signature = {
+        MODEL: "TS011F",
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=266
+            # device_version=1
+            # input_clusters=[0, 3, 4, 5, 6, 1026, 1794, 2820, 57344, 57345]
+            # output_clusters=[10, 25]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_PLUG_IN_UNIT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    0x0402,  # unknown meaning, it is not temperature
```

Comment:
add the `from zigpy.zcl.clusters.measurement import TemperatureMeasurement` (between the `TemperatureMeasurement` and the `Metering`) and use the `TemperatureMeasurement.cluster_id`